internal/resources/ekscluster/flatten_test.go

```diff
@@ -0,0 +1,1460 @@
+/*
+Copyright 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package ekscluster
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	eksmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/ekscluster"
+)
+
+func TestFlattenCluterSpec(t *testing.T) {
```

Comment:
Same goes for the flatten tests, dividing the eks resource into two and taking the tests separately will make it more organised. Also, let's avoid having more than 1000 lines of code in one file.

Reply:
done